README.md

```diff
@@ -22,3 +35,17 @@ Everything you ever wanted to know about the CRA and its implementation.
 [related FAQ]: https://github.com/orcwg/cra-hub/blob/main/faq.md#important-and-critical-product-categories
 [related FAQ issues]: https://github.com/orcwg/cra-hub/issues?q=is:issue%20state:open%20label:%22Critical%20and%20Important%20Products%22%20label:FAQ
 
+## How to Contribute
+
+ORC has mobilised a dedicated community effort to support open source stakeholders. Recognising the unique challenges faced by developers, maintainers, and organisations, we are collaboratively developing a suite of resources designed to demystify the CRA and provide actionable guidance. Our key initiatives, each crafted through community input, empower stakeholders to achieve informed and effective compliance.
```

Comment:
Are we using both "ORC" and "ORCWG" in different places. I think we should streamline on "orcwg" as in the domain.

Reply:
Thanks for flagging this. We use _Open Regulatory Compliance (ORC) Working Group_ on first mention, then just _ORC_ after that. The repo and domain name use _ORCWG_, which is fine in those contexts, but it doesn’t work as well in regular text, so we try to avoid it outside of asset references.

Reply:
This is a readme, not a marketing pitch. Can we be factual and concrete in terms of what someone has to do to contribute here?